app/src/main/res/values/strings.xml

```diff
@@ -64,4 +64,18 @@
     <string name="skills_button">Skills</string>
     <string name="cpr_rate_button_start">START</string>
     <string name="cpr_rate_button_stop">STOP</string>
+    <string name="save_changes">Save changes</string>
+    <string name="changes_saved">Changes saved</string>
+    <string name="invalid_field_msg">ERROR : Some field aren\'t fill correctly</string>
+    <string name="privacy_policy_not_acceptes">ERROR : You must accepted our privacy policy first</string>
+    <string name="info_saved">Your information have been saved</string>
+    <string name="medical_info_prefs"> medicalInfoPrefs</string>
+    <string name="medical_info_key">MEDICAL_INFO_KEY</string>
+    <string name="privacy_policy">Privacy policy</string>
+    <string name="medical_info_privacy_policy">
+        We know how medical information are important and we take care of your privacy. Indeed your medical data are only stored on your phone.\n In case of an emergency only your helper will received your info so he/she can help you in the best condition. After the intervention, your info will then be deleted from your helper\'s phone and you will again be the only owner of your information</string>
+    <string name="condition_help_msg">This Field is dedicated to any illness or diseases that a caregiver should know in case of emergency</string>
+    <string name="treatment_help_msg">This Field is dedicated to any medication your are taking that a caregiver should know in case of emergency</string>
```

Comment:
```suggestion
    <string name="treatment_help_msg">This Field is dedicated to any medication you are taking that a caregiver should know in case of emergency</string>
```

Reply:
Damn! I'm a bit ashamed  of all these grammatical errors, sorry